### ISSUE TITLE
Accessibility, elements for customising custom variables when creating template are unlabeled for screenreader

### ISSUE THREAD

**Issue by Jonathans859** [joplin/plugin-templates#121]
Hi, when I create a template with custom variables I have to fill in at creation, the elements to fill in are not labeled for my screenreader NVDA.
e.g., let's say I have this header:
---
name: text
template_title: MW - {{name}} - {{date}}
has_vip: boolean
do_everything: boolean
zone2: boolean
event: boolean
---

upn creation when tabbing through the pop up dialog, I only hear
edit  blank
combo box  Yes  collapsed
combo box  Yes  collapsed
combo box  Yes  collapsed
combo box  Yes  collapsed
Joplin  document  ￼
OK  button
